backend_nestjs/dining-service/src/table-orders/services/table-orders.service.ts

```diff
@@ -57,7 +58,7 @@ export class TableOrdersService {
 
   async addOrderingLineToTableOrder(tableOrderId: string, addMenuItemDto: AddMenuItemDto): Promise<TableOrder> {
     const tableOrder: TableOrder = await this.findOne(tableOrderId);
-
+    console.log(tableOrder);
```

Comment:
Can you add something to the log to say it is about adding item

Reply:
that was for debugging xD